www/index.html

```diff
@@ -487,10 +491,17 @@ <h2 data-i18n="configure-title">Configuration</h2>
                             <br /> Scanning for archives... Please wait <img src="img/spinner.gif" alt="Please wait..." />
                         </div>
                         <div id="chooseArchiveFromLocalStorage" style="display: none;">
-                            <br /> Please select the archive you want to use : <select id="archiveList"
-                                class="form-control"></select>
-                            <br /><button type="button" class="btn btn-light" id="btnRescanDeviceStorage">Rescan</button>
-                            Rescans your SD Cards and internal memory
+                            <div style="float: left;display: none;" id="fileCountDisplay">
```

Comment:
You shouldn't use these kinds of inline styles, and especially it shouldn't be necessary to float elements other than arguably some buttons. Use the correct Bootsrap classes for UI elements instead, if you're having difficulty getting alignment right. You have to assign classes to containing divs, e.g. `<div class="row">` to contain UI elements that should be aligned in the same row (rows might need to be wrapped in `<div class="container">`). I was looking at this because I wanted to ask you to add some vertical spacing in the screenshot below where the green line is. If you wrap textual elements in `<p>` rather than using `<br />`, they should get the appropriate vertical spacing (I realize there is a lot of legacy use of `<br />` in the original HTML, which I didn't write).

![image](https://github.com/kiwix/kiwix-js/assets/4304337/96e3cf69-fda3-4c80-acad-1d196ac7e719)